src/DataManagement/Entries/EntryManager.cs

```diff
@@ -84,42 +92,43 @@ public bool Edit(int id, IEntry newEntry)
                 ExceptionHandler.Log("Edited entry id: " + id + " in database but couldn't find it locally");
                 return false;
             }
-            temp.Edit(newEntry);
+            temp.Edit(updatedEntry);
             return true;
         }
 
         public bool Remove(IEntry entry)
         {
-            if (!EntryDbUpdater.Remove(entry.Id, UserId, EntryList.EntryType)) return false;
-            var temp = EntryList.FirstOrDefault(x => x.Id == entry.Id);
-            if (temp is null)
-            {
-                ExceptionHandler.Log("Removed entry id: " + entry.Id + " from database but couldn't find it locally");
-                return false;
-            }
-            EntryList.Remove(temp);
-            return true;
+            return Remove(entry.Id);
         }
+
         public bool Remove(int id)
         {
-            if (!EntryDbUpdater.Remove(id, UserId, EntryList.EntryType)) return false;
+            if (!EntryDatabase.Delete(id, UserId, EntryList.EntryType))
+            {
+                return false;
+            }
+
             var temp = EntryList.FirstOrDefault(x => x.Id == id);
+
             if (temp is null)
             {
                 ExceptionHandler.Log("Removed entry id: " + id + " from database but couldn't find it locally");
                 return false;
             }
+
             EntryList.Remove(temp);
             return true;
         }
 
         public bool RemoveAll(IEntryList entryList)
         {
-            var idList = entryList.Select(va => va.Id).ToArray();
-            if (!EntryDbUpdater.RemoveAll(idList, entryList.EntryType))
+            var idArray = entryList.Select(va => va.Id).ToArray();
```

Comment:
Random question, why "va" is used? :D

Reply:
Have no idea how that's in here, will add a fix for these :D